xbmc/utils/XMLUtils.cpp

```diff
@@ -306,6 +306,12 @@ TiXmlNode* XMLUtils::SetFloat(TiXmlNode* pRootNode, const char *strTag, float va
   return SetString(pRootNode, strTag, strValue);
 }
 
+TiXmlNode* XMLUtils::SetDouble(TiXmlNode* pRootNode, const char *strTag, double value)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types